### PR TITLE
Feature/getBalance

### PR DESCRIPTION
**Pull request by bzuker**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/blockcoders/near-rpc-providers/blob/main/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: src/responses.ts [hunk 1]
```diff
@@ -27,3 +27,13 @@ export interface StatusRpcResponse {
...
+
+export interface GetBalanceRpcResponse {
+  amount: string
+  locked: string
+  code_hash: string
+  storage_usage: number
+  storage_paid_at: number
+  block_height: number
+  block_hash: string
+}
```
Comment (jarcodallo) [inline src/responses.ts:39]: I don't think we need to map the whole interface. Having amount is enough.

File: src/near-rpc-provider.ts [hunk 2]
```diff
@@ -154,10 +162,42 @@ export class NearRpcProvider extends JsonRpcProvider {
...
+      case 'getBalance':
+        return this._internalGetBalance(method, params)
       default:
         return super.perform(method, params)
     }
   }
+
+  _getAddress(addressOrName: string | Promise<string>): Promise<string> {
+    return Promise.resolve(addressOrName)
+  }
```
Comment (jarcodallo) [inline src/near-rpc-provider.ts:175]: perfect, we need this for other calls 🚀

File: src/near-rpc-provider.ts [hunk 3]
```diff
@@ -154,10 +162,42 @@ export class NearRpcProvider extends JsonRpcProvider {
...
+      finality: 'final' as const,
+    }
+
+    const getBalanceParams: GetBalanceParams = {
+      ...defaultParams,
+      ...params,
+      account_id: params.address,
+    }
+
+    validateGetBalanceParams(getBalanceParams)
```
Comment (jarcodallo) [inline src/near-rpc-provider.ts:189]: we don't need to validate the params. We just want to get the balance from an account. this function `_internalGetBalance` just need to received the address and that's it. Then build the query to get the balance and return a bignumber

**Review by jarcodallo — COMMENTED** [review]
So far looks good, but let's just parse the data that we need to get the balance.

**Review by AnnRiera — APPROVED** [review]
LGTM!

**Review by jarcodallo — APPROVED** [review]

**Pull request merged**